code/game/objects/items.dm

```diff
@@ -572,3 +574,6 @@ obj/item/proc/item_action_slot_check(slot, mob/user)
 
 /obj/item/proc/is_sharp()
 	return sharpness
+
+/obj/item/proc/can_dismember()
+	return sharpness && w_class >= 3
```

Comment:
It should be a separate function/var though. In the same way that a wirecutter is a substitute tool for clamping bleeders, but isn't always successful.